study-builder/studies/rarex/study.conf

```diff
@@ -1026,9 +1042,82 @@
       },
       "action": {
         "type": "ACTIVITY_INSTANCE_CREATION",
-        "activityCode": "PARENTAL_QUALITY_OF_LIFE"
+        "activityCode": "CHILD_QUALITY_OF_LIFE"
+      },
+      "preconditionExpr": """!user.studies["rarex"].forms["ADD_PARTICIPANT"].questions["PARTICIPANT_RELATIONSHIP"].answers.hasOption("SELF")
+        && (
+          ( user.studies["rarex"].forms["ADD_PARTICIPANT"].questions["PARTICIPANT_COUNTRY"].answers.hasOption("US")
+            && (
+              ( user.studies["rarex"].forms["ADD_PARTICIPANT"].questions["PARTICIPANT_STATE"].answers.hasOption("AL")
+                && user.profile.age() < 19
```

Comment:
We likely need to do this by "age in locale" (what I have now) rather than hardcoded `>= 18`. Still waiting on final decision here.